app/views/user_mailer/shared/_in_person_ready_to_verify.html.erb

```diff
@@ -35,44 +38,176 @@
   </tr>
 </table>
 
-<div class="border-1px border-primary-light radius-lg padding-4 margin-bottom-4">
+<%# Enhanced IPP Only - What to bring section %>
+<% if @is_enhanced_ipp %>
+  <section class="border-1px radius-lg border-primary-light padding-4 margin-bottom-4">
+    <%# What to bring to the Post Office %>
+    <h2 class="margin-bottom-2"><%= t('in_person_proofing.headings.barcode_what_to_bring') %></h2>
+    <p class="margin-bottom-2"><%= t('in_person_proofing.body.barcode.what_to_bring') %></p>
+
+    <%# Option 1: Bring a Real ID %>
+    <h3 class="margin-bottom-2"><%= t('in_person_proofing.process.eipp_bring_id.heading') %></h3>
+    <div>
+      <table>
+        <tbody>
+          <tr>
+            <td align="left" width="110.46px">
+              <%= image_tag(
+                    asset_url('idv/real-id.svg'),
+                    width: 110,
+                    height: 80,
+                    alt: t('in_person_proofing.process.eipp_bring_id.image_alt_text'),
+                    role: 'img',
+                    class: 'margin-bottom-3',
+                  ) %>
+            </td>
+            <td width="24px">
+            </td>
+            <td>
+              <p class="margin-bottom-3"><%= t('in_person_proofing.process.eipp_bring_id.info') %></p>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+    <hr class="margin-bottom-3 border-primary-light border-top-width-0 margin-top-0">
+
+    <%# Option 2: Bring a standard State ID... %>
+    <h3 class="margin-bottom-2"><%= t('in_person_proofing.process.eipp_what_to_bring.heading') %></h3>
+    <p class="margin-bottom-3"><%= t('in_person_proofing.process.eipp_what_to_bring.info') %></p>
+
+    <%# A. State ID + Passport %>
+    <table class="margin-bottom-3">
+      <tbody>
+        <tr>
+          <td align="left" width="110.46px">
```

Comment:
`td` `width` is deprecated, and I don't think it expected units to be provided (hard to verify since the spec doesn't document the attribute anymore)

Reply:
[PR# 10839](https://github.com/18F/identity-idp/pull/10839) handles this comment.